### ISSUE TITLE
[Bug]: Prevent blocking on `start_session` and `end_session`

### ISSUE THREAD

**Issue by areibman** [AgentOps-AI/agentops#535]
### 📦 Package Version

0.3.18

### 🔎 Describe the Bug

Currently, `start_session` and `end_session` are blocking on the main thread. This is because start_session fetches a JWT which can take some time. We can move this to `agentops.init` to speed things up.

Additionally, `end_session` runs a flush. We can probably make this more of a "fire and forget" style signal. That is, when the `SessionExporter` sees an end_session event, it doesn't need to trigger a flush. We can just reserve the flush for SIGINT and program exit.

### 🤝 Contribution

- [ ] Yes, I'd be happy to submit a pull request with these changes.
- [ ] I need some guidance on how to contribute.
- [X] I'd prefer the AgentOps team to handle this update.

**Comment by Dwij1704** [comment]
This is true, we should bring this back on roadmap, it awaits for JWT which slows down the execution
